nnf-container-example.yaml

```diff
@@ -17,9 +17,10 @@ data:
           spec:
             containers:
               - name: nnf-container-example
-                image: ghcr.io/nearnodeflash/nnf-container-example:latest
+                image: ghcr.io/nearnodeflash/nnf-container-example:0.0.0.14-496e
```

Comment:
Do we want latest here?

Reply:
'latest' is gone. I'm switching this repo to use docker/metadata-action.

Reply:
But we probably want 'master' here.